markdown_it/tree.py

```diff
@@ -183,6 +195,23 @@ def _set_children_from_tokens(self, tokens: Sequence[Token]) -> None:
             )
             child._set_children_from_tokens(nested_tokens[1:-1])
 
+    def pretty(
```

Comment:
Should we print closing tags or add the slash character to make the tags self-closing if there's no content? That would make the output more or less valid XML and wouldn't confuse highlighters, parsers etc.

Reply:
I'm not adding closing tags as the indentation denotes the nesting and this would make it overly verbose and less readable (which is the main point of this function). I could add self-closing, but TBH the highlighting in tests/test_tree/test_pretty.xml seems to be fine